Code/GraphMol/Aromaticity.cpp

```diff
@@ -686,41 +695,60 @@ int setAromaticity(RWMol &mol) {
   VECT_INT_VECT brings;
   RingUtils::convertToBonds(cRings, brings, mol);
 
-  // make the neighbor map for the rings
-  // i.e. a ring is a neighbor a another candidate ring if
-  // shares at least one bond
-  // useful to figure out fused systems
-  INT_INT_VECT_MAP neighMap;
-  RingUtils::makeRingNeighborMap(brings, neighMap, maxFusedAromaticRingSize);
-
-  // now loop over all the candidate rings and check the
-  // huckel rule - of course paying attention to fused systems.
-  INT_VECT doneRs;
-  int curr = 0;
-  int cnrs = rdcast<int>(cRings.size());
-  boost::dynamic_bitset<> fusDone(cnrs);
-  INT_VECT fused;
-  while (curr < cnrs) {
-    fused.resize(0);
-    RingUtils::pickFusedRings(curr, neighMap, fused, fusDone);
-    applyHuckelToFused(mol, cRings, brings, fused, edon, neighMap, narom, 6);
-
-    int rix;
-    for (rix = 0; rix < cnrs; rix++) {
-      if (!fusDone[rix]) {
-        curr = rix;
+  if (!includeFused) {
+    // now loop over all the candidate rings and check the
+    // huckel rule - skipping fused systems
+    INT_INT_VECT_MAP neighMap;
+    for (unsigned int ri = 0; ri < cRings.size(); ++ri) {
+      INT_VECT fused;
+      fused.push_back(ri);
+      applyHuckelToFused(mol, cRings, brings, fused, edon, neighMap, narom, 6);
+    }
+  } else {
+    // make the neighbor map for the rings
+    // i.e. a ring is a neighbor a another candidate ring if
+    // shares at least one bond
+    // useful to figure out fused systems
+    INT_INT_VECT_MAP neighMap;
+    RingUtils::makeRingNeighborMap(brings, neighMap, maxFusedAromaticRingSize);
+
+    // now loop over all the candidate rings and check the
+    // huckel rule - of course paying attention to fused systems.
+    INT_VECT doneRs;
+    int curr = 0;
+    int cnrs = rdcast<int>(cRings.size());
+    boost::dynamic_bitset<> fusDone(cnrs);
+    INT_VECT fused;
+    while (curr < cnrs) {
+      fused.resize(0);
+      RingUtils::pickFusedRings(curr, neighMap, fused, fusDone);
+      applyHuckelToFused(mol, cRings, brings, fused, edon, neighMap, narom, 6);
+
+      int rix;
+      for (rix = 0; rix < cnrs; rix++) {
+        if (!fusDone[rix]) {
+          curr = rix;
+          break;
+        }
+      }
+      if (rix == cnrs) {
         break;
       }
     }
-    if (rix == cnrs) {
-      break;
-    }
   }
 
   mol.setProp(common_properties::numArom, narom, true);
 
   return narom;
 }
 
+}  // end of anonymous namespace
+
+int setAromaticity(RWMol &mol) { return aromaticityHelper(mol, 0, 0, true); }
```

Comment:
If we are going to potentially support several aromaticity models, I think I might prefer something like:

```
namespace AromaticityModel {
const unsigned int Default = 0; // Global setting or RDKit
const unsigned int RDKit = 1;
const unsigned int Simple = 2;
} // or enum...

int setAromaticity(RWMol &mol, 
                            unsigned int aromaticityModel=AromaticityModel::Default) {
  case ...
}
```

This will allow flexibility in the future to add MDL models/etc and have a way to set the default model once and have it propagate everywhere.  I'm not sure we would need to implement this initially, but leaving space for it is useful.